spacewalk/config/usr/share/man/man5/rhn.conf.5

```diff
@@ -215,13 +215,20 @@ then the job will be queued.
 2
 
 .TP
-.B "taskomatic.maxmemory" (integer)
+.B "taskomatic.java.maxmemory" (integer)
```

Comment:
After applying following to rhn.conf
taskomatic.java.initmemory=512
taskomatic.java.maxmemory=1192

I can see following:
root     23361  0.0  0.0  21180   680 ?        Sl   09:34   0:00 /usr/bin/taskomaticd /usr/share/rhn/config-defaults/rhn_taskomatic_daemon.conf wrapper.maxmemory=2000 wrapper.java.initmemory=512 wrapper.java.maxmemory=1192 wrapper.pidfile=/var/run//taskomatic.pid wrapper.daemonize=TRUE

I guess having wrapper.maxmemory=2000 and wrapper.java.maxmemory=1192 together won't work well.

Putting following into rhn.conf (removed .java):
taskomatic.initmemory=512
taskomatic.maxmemory=1192

which results in:
root     23660  0.0  0.0  21176   680 ?        Sl   09:38   0:00 /usr/bin/taskomaticd /usr/share/rhn/config-defaults/rhn_taskomatic_daemon.conf wrapper.maxmemory=2000 wrapper.initmemory=512 wrapper.maxmemory=1192 wrapper.pidfile=/var/run//taskomatic.pid wrapper.daemonize=TRUE

where I'd say latter takes precedence so it should be OK.

Which is weird as in /usr/share/rhn/config-defaults/rhn_taskomatic_daemon.conf there's .java in config option name
wrapper.java.initmemory=256
wrapper.java.maxmemory=1024

I guess I need to investigate a bit more :)